Code/Tools/AssetProcessor/native/tests/test_main.cpp

```diff
@@ -44,7 +45,10 @@ int main(int argc, char* argv[])
     }
     
     bool ranUnitTests;
+
+    AZ::Environment::Create(nullptr);
```

Comment:
The AZ::Environment is normally created lazily via a call to [AZ::Environment::GetInstance](https://github.com/o3de/o3de/blob/development/Code/Tools/AzTestRunner/Platform/Android/platform_android.cpp#L300) instead of a `call to `AZ::Environment::Create`:

Reply:
This was fixed upstream by 346a414aff6a79b4651337b473c82f8c924aa749 anyway, so I've dropped this commit from the PR.